docs/content/installation/using-the-jwt-token-docker-secret.md

```diff
@@ -18,51 +18,150 @@ This document explains how to use the NGINX Plus Ingress Controller image from t
 * For NGINX Ingress Controller, you must have the NGINX Ingress Controller subscription -- download the NGINX Plus Ingress Controller (per instance) JWT access token from [MyF5](https://my.f5.com).
 * To list the available image tags using the Docker registry API, you will also need to download the NGINX Plus Ingress Controller (per instance) certificate (`nginx-repo.crt`) and the key (`nginx-repo.key`) from [MyF5](https://my.f5.com).
 
+## Order of steps to pull NGINX Ingress controller from F5  registry
+
+1. Decide on what NGINX Ingress controller image you want to use. [NGINX Ingress controller images](https://docs.nginx.com/nginx-ingress-controller/technical-specifications/#images-with-nginx-plus "Available NGINX Ingress controller images")
+2. Log into MyF5 portal. [MyF5 portal login](https://myf5.com/ "MyF5 portal login")
```

Comment:
Perhaps give more description here.  Something like:  You'll find the token under the "Subscription Detail" section of your subscription.